setup.py

```diff
@@ -64,7 +64,6 @@ def run_setup():
 
         install_requires=[
             'six',
-            'jsonconversion'
```

Comment:
What are the implications of this? Does it break something else? Why was it added in the first place?

Reply:
Discussed offline, I'll create a new commit and comment detail there.